### PR TITLE
Complete Remove File server definition in Azure Rest API

### PR DESCRIPTION
**Pull request by wzhang373**
This change Completely remove file server definitions in the rest apis.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, deprecate old apis.
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.. ASAP
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. Jun, 2021
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [X] SDK of Python
      - [X] SDK of Java
      - [X] SDK of Js
      - [X] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [X] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [X] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [x] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by wzhang373** [comment]
Can anyone have a look and approve my change for merge conflict?

**Review by JeffreyRichter — APPROVED** [review]

**Comment by jianyexi** [comment]
@ArcturusZhang  could you approve the SDK breaking change

File: specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-05-01/BatchAI.json [hunk 1]
```diff
@@ -941,112 +941,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BatchAI/workspaces/{workspaceName}/fileServers/{fileServerName}": {
```
Comment (chiragg4u) [inline specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-05-01/BatchAI.json:944]: >fileServers [](http://example.com/codeflow?start=127&length=11)

Please follow the deprecation process @ [https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html](https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html)

If it's already followed please share the evidences for the same for approving.

**Comment by jianyexi** [comment]
@wzhang373 any update ?

**Comment by wzhang373** [comment]
Which one? This may already be checked in with another pr.
________________________________

**Comment by jianyexi** [comment]
> Which one? This may already be checked in with another pr.
> […](#)
> ________________________________ From: msftbot[bot] ***@***.***> Sent: Sunday, July 18, 2021 9:02:40 AM To: Azure/azure-rest-api-specs ***@***.***> Cc: Weiguo Zhang ***@***.***>; Mention ***@***.***> Subject: Re: [Azure/azure-rest-api-specs] Complete Remove File server definition in Azure Rest API (#14629) Hi, @wzhang373 Weiguo Zhang FTE<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fwzhang373&data=04%7C01%7Cwzhang%40exchange.microsoft.com%7C324c30e04fd449e2183708d94a05788f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637622209657664362%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=RCPkO%2FxyWfDEVycF9p5ePUPPEHXyqUKgeuOhL3bbEB4%3D&reserved=0>. Your PR has no update for 14 days and it is marked as stale PR. If no further update for over 14 days, the bot will close the PR. If you want to refresh the PR, please remove no-recent-activity label. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F14629%23issuecomment-882079263&data=04%7C01%7Cwzhang%40exchange.microsoft.com%7C324c30e04fd449e2183708d94a05788f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637622209657664362%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=vZZEm0iXn4LK9L39xv55zx%2BYAGu%2BEHH5MIs4ELY1zgQ%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAKJXGAL3MJJPHOAMOF4ZTYDTYL3KBANCNFSM455TLHHQ&data=04%7C01%7Cwzhang%40exchange.microsoft.com%7C324c30e04fd449e2183708d94a05788f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637622209657674354%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=1Spx84KHVtvFX%2F232d4Ts8WMaa6f2u3OZrf3qPVy7G8%3D&reserved=0>.

So we can  close this one ?

**Pull request closed without merging**